compute_endpoint/globus_compute_endpoint/endpoint/config/utils.py

```diff
@@ -147,16 +147,19 @@ def load_user_config_schema(endpoint_dir: pathlib.Path) -> dict | None:
 
 def _validate_user_opts(user_opts: dict, schema: dict | None) -> None:
     """Validates user config options, optionally against a JSON schema."""
-    import jsonschema  # Only load package when called by EP manager
 
-    if "parent_config" in user_opts:
-        raise ValueError(
-            "'parent_config' is a reserved word and cannot be passed in via user config"
-        )
+    for reserved_word in ["parent_config", "user_runtime"]:
+        if reserved_word in user_opts:
```

Comment:
For easier referencing (for example, a single-point of authority, and for tests), consider defining this now-more-than-one-item-list.  Perhaps:
```python
# utils.py
import ...
import ...

RESERVED_CONFIGURATION_KEYS = (
    "parent_config",
    "user_runtime",
)
```

Then can add a test to verify that this list hasn't changed (`assert expected_set == set(RESERVED_CONFIGURATION_KEYS)`), and similarly reference it in `test_validate_user_opts_reserved_words`.  "Single point of authority."